draft-ietf-tls-esni.md

```diff
@@ -874,9 +892,9 @@ this problem by flushing any DNS or ECHConfig state upon changing networks.
 
 Optional record digests may be useful in scenarios where clients and
 client-facing servers do not want to reveal information about the client-facing
-server in the "encrypted_client_hello" extension. In such settings, clients send
+server in the "ech" extension. In such settings, clients send
 either an empty config_digest or a randomly generated config_digest in the
```

Comment:
(I lied, here's the one place we did not hyphenate "randomly generated".)

Reply:
Ack. "randomly generated" only appears at one point in the text, which is not related to this change.